amf-cli/shared/src/test/resources/validations/async20/validations/binding-wrong-key.yaml

```diff
@@ -0,0 +1,35 @@
+asyncapi: 2.5.0
+info:
+  title: API with GooglePubSub Message Binding Validation
+  version: 1.0.0
+servers:
+  googlePubSubServer:
+    url: localhost:9092
+    protocol: googlepubsub
+    description: A Google Pub/Sub server for messaging.
+    extraKey: an extra key
+channels:
+  userEventChannel:
+    subscribe:
+      message:
+        $ref: '#/components/messages/UserEvent'
+components:
+  messages:
+    UserEvent:
+      summary: A message for user events with validation errors.
+      payload:
+        type: object
+        properties:
+          userId:
+            type: string
+            description: Unique identifier for the user.
+          action:
+            type: string
+            description: Description of the user action.
+      bindings:
+        googlepubsub:
+          attributes:
+            exampleAttribute: "exampleValue"
+          wrongOrderingKey: "userEventOrder"
```

Comment:
Try also adding extra keys to new nested complex models, like `schema` in this case

Reply:
added schema